### PR TITLE
Paradise Ontology permanent id request

### PR DESCRIPTION
**Pull request by beyzayaman**
Hi! I'd like to get a permanent ID for our ontology which sits at 

http://ontologies.adaptcentre.ie/paradise/index-en.html

Kind Regards,
Beyza.

### CODE REVIEW

File: paradise/.htaccess [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+RewriteEngine On
+#Change the path to the folder here
+RewriteBase /paradise 
+
+# Rewrite rule to serve HTML content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^$ index-en.html [R=303,L]
```
Comment (dgarijo) [inline paradise/.htaccess:21]: Remove the rewrite base (row 14) and replace lines 21, 25 etc with the links instead please.
For example, here replace `index-en.html` with  `http://ontologies.adaptcentre.ie/paradise/index-en.html`

File: paradise/readme.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+## The Paradise Ontology Repository
+
+Ontology
```
Comment (TallTed) [inline paradise/readme.md:3]:
```suggestion
### Ontology
```

File: paradise/readme.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+* Doc http://ontologies.adaptcentre.ie/paradise/
+* Turtle http://ontologies.adaptcentre.ie/paradise/ontology.ttl
+* RDF/XML http://ontologies.adaptcentre.ie/paradise/ontology.xml
+* JSON-LD http://ontologies.adaptcentre.ie/paradise/ontology.json
+* N-Triples http://ontologies.adaptcentre.ie/paradise/ontology.nt
+
+
+Maintainers:
+
+Beyza Yaman (beyza.yaman@adaptcentre.ie)
```
Comment (TallTed) [inline paradise/readme.md:14]:
```suggestion
### Maintainers

* Beyza Yaman (beyza.yaman@adaptcentre.ie)
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Small changes for clarity

**Comment by beyzayaman** [comment]
Thank you for your suggestions and feedback. I pushed the changes again.

**Pull request merged**